float64[] kotiVRadianih
float64[] razdalje
geometry_msgs/Point[] tocke
time cas 
